src/collective/workspace/tests/test_workspace.py

```diff
@@ -44,3 +68,20 @@ def test_remove_from_team(self):
             user=self.user1.getId()
         )
         self.assertNotIn(self.user1.getId(), list(self.ws.members))
+
+    def test_add_guest_to_team(self):
+        self.ws.add_to_team(
+            user=self.user1.getId(), groups=['Guests']
+        )
+        self.assertIn(self.user1.getId(), list(self.ws.members))
+
+    def test_guest_has_no_team_member_role(self):
+        self.ws.add_to_team(
+            user=self.user1.getId(), groups=['Guests']
+        )
+        self.reset_annotations()
```

Comment:
Note: This is going to be outdated if this #16 get merged because `add_to_team` notiies `TeamMemberAddedEvent`, which causes a cache purge.